### DISCUSSION TITLE
Anyone have idea on how to install on Proton Persona?

### DISCUSSION THREAD

**Discussion by deepblue1004** [xeon1989/Proton-X50-APK-Installer#6] [Q&A] [answered]
I tried the step in readme.md on my Proton Persona 2022 premium, but it seems doesn't work, is there anyone already succeed?

**Comment by xeon1989** [comment]
The script is designed for GKUI19 running on Proton X50. It is not compatible with Proton Persona 2022.

**Reply by Unknown0623** [reply]
Any others way to install on persona premium 2022?

**Comment by deepblue1004** [comment, marked as answer]
##Found the solution

Just uninstall spotify and reinstall it, when prompt for notification permission, press `back` button, will unlock Android native `setting` page.
Go to `storage` section to read apk file inside pendrive. 
Done √
